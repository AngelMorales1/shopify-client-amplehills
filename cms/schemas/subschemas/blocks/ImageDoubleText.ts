import image from './../image';
import color from './../color';
import portableText from './../portableText';

export default {
  type: 'object',
  name: 'imageDoubleText',
  title: 'Image Double Text',
  fieldsets: [
    {
      name: 'options',
      title: 'Options',
      options: { collapsible: true, collapsed: true }
    },
    {
      name: 'buttons',
      title: 'Buttons',
      options: { collapsible: true, collapsed: true }
    },
    {
      name: 'image',
      title: 'Image',
      options: { collapsible: true, collapsed: true }
    },
    {
      name: 'text1',
      title: 'Text Block 1',
      options: { collapsible: true, collapsed: true }
    },
    {
      name: 'text2',
      title: 'Text Block 2',
      options: { collapsible: true, collapsed: true }
    }
  ],
  fields: [
    {
      name: 'title',
      title: 'Title',
      type: 'string'
    },
    color({
      name: 'backgroundColor',
      title: 'Background Color'
    }),
    {
      name: 'title1',
      title: 'Title 1',
      type: 'string',
      fieldset: 'text1'
    },
    {
      name: 'subtitle',
      title: 'Subtitle',
      type: 'string',
      fieldset: 'text1'
    },
    color({
      name: 'subtitleColor',
      title: 'Subtitle Color',
      limit: ['peach', 'blue'],
      fieldset: 'text1'
    }),
    portableText({
      name: 'text1',
      title: 'Text 1',
      fieldset: 'text1'
    }),
    {
      name: 'title2',
      title: 'Title 2',
      type: 'string',
      fieldset: 'text2'
    },
    {
      name: 'subtitle2',
      title: 'Subtitle 2',
      type: 'string',
      fieldset: 'text2'
    },
    color({
      name: 'subtitle2Color',
      title: 'Subtitle Color',
      limit: ['peach', 'blue'],
      fieldset: 'text2'
    }),
    portableText({
      name: 'text2',
      title: 'Text 2',
      fieldset: 'text2'
    }),
    image({
      name: 'image',
      title: 'Image',
      fieldset: 'image'
    }),
    {
      name: 'buttonLabel',
      title: 'Button Label',
      fieldset: 'buttons',
      type: 'string'
    },
    color({
      name: 'buttonColor',
      title: 'Button Color',
      fieldset: 'buttons',
      limit: ['peach', 'navy']
    }),
    {
      name: 'buttonLink',
      title: 'Button Link',
      fieldset: 'buttons',
      type: 'url',
      validation: Rule => Rule.uri({ allowRelative: true })
    },
    {
      name: 'textLinkLabel',
      title: 'Text Link Label',
      fieldset: 'buttons',
      type: 'string'
    },
    {
      name: 'textLink',
      title: 'Text Link',
      fieldset: 'buttons',
      type: 'url',
      validation: Rule => Rule.uri({ allowRelative: true })
    },
    {
      name: 'drip',
      title: 'Has Drip?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'upperDrip',
      title: 'Has Upper Drip?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'reverseX',
      title: 'Reverse X Arrangement?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'reverseY',
      title: 'Reverse Y Arrangement?',
      fieldset: 'options',
      type: 'boolean'
    },
  ]
};
