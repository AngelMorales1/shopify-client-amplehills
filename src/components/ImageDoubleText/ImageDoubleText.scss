@import 'settings';

.ImageDoubleText {
  &--white {
    @extend .bg-white;
  }

  &--yellow {
    @extend .bg-yellow;
  }

  &--light-yellow {
    @extend .bg-light-yellow;
  }

  &--mustard {
    @extend .bg-deep-yellow;
  }

  &--blue {
    @extend .bg-sky-blue;
  }

  &--light-turquoise {
    @extend .bg-light-turquoise;
  }

  &--pink {
    @extend .bg-light-pink;
  }

  &--light-pink {
    @extend .bg-baby-pink;
  }

  &--light-peach {
    @extend .bg-light-peach;
  }

  &--light-orange {
    @extend .bg-light-orange;
  }

  &--peach-pink {
    @extend .bg-peach-pink;
  }

  &--yellow-green {
    @extend .bg-yellow-green;
  }

  &--light-yellow-green {
    @extend .bg-light-yellow-green;
  }

  &__title {
    font-size: $font-6;

    @media ($medium-up) {
      font-size: 3.5rem;
    }
  }

  &--container {
    flex-direction: column;

    @media ($medium-up) {
      flex-direction: row;
      width: 82.5rem;
    }

    .ImageDoubleText--image-container {
      width: 65%;
      min-width: spacing(100);
      margin-top: spacing(-24);
      margin-left: spacing(-55);

      @media ($medium-up) {
        margin-bottom: spacing(16);
        margin-top: spacing(-40);
        margin-left: spacing(-84);
      }
    }

    .ImageDoubleText--text-container {
      padding: spacing(8);

      @media ($medium-up) {
        padding: spacing(12);
      }
    }
  }

  &--container--reverse-x {
    .ImageDoubleText--image-container {
      margin-left: 0;
      margin-right: spacing(-55);

      @media ($medium-up) {
        order: 2;
        margin-left: 0;
        margin-right: spacing(-84);
      }
    }

    .ImageDoubleText--text-container {
      padding: spacing(8);

      @media ($medium-up) {
        order: 1;
        padding: spacing(12);
      }
    }
  }

  &--container--reverse-y {
    .ImageDoubleText--image-container {
      margin-bottom: spacing(-24);
      margin-top: 0;
      order: 1;

      @media ($medium-up) {
        order: initial;
        margin-top: spacing(16);
        margin-bottom: spacing(-40);
      }
    }

    .ImageDoubleText--text-container {
      order: 0;

      @media ($medium-up) {
        order: initial;
      }
    }
  }

  &--container--reverse-xy {
    .ImageDoubleText--image-container {
      @media ($medium-up) {
        order: 1;
      }
    }

    .ImageDoubleText--text-container {
      @media ($medium-up) {
        order: 0;
      }
    }
  }
}
