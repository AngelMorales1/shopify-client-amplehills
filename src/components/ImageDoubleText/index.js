import React from 'react';
import PropTypes from 'prop-types';
import marked from 'marked';
import cx from 'classnames';
import get from 'utils/get';
import contentfulImgUtil from 'utils/contentfulImgUtil';
import imageModel from 'models/imageModel';

import styles from './ImageDoubleText.scss';
import { Button } from 'components/base';

const ImageDoubleText = ({ block, z, setRef }) => {
  const fields = get(block, 'fields', {});
  const colorClass = `ImageDoubleText--${get(
    fields,
    'backgroundColor',
    'white'
  )}`;
  const reverseArrangementX = get(fields, 'reverseArrangementX', false);
  const reverseArrangementY = get(fields, 'reverseArrangementY', false);
  const smallTitle = get(fields, 'smallTitle', '');
  const smallTitleColor = get(fields, 'smallTitleColor', 'madison-blue');
  const title = get(fields, 'title', '');
  const title1 = get(fields, 'title1', '');
  const text1 = get(fields, 'text1', '');
  const title2 = get(fields, 'title2', '');
  const text2 = get(fields, 'text2', '');
  const buttonLabel = get(fields, 'buttonLabel', '');
  const buttonLink = get(fields, 'buttonLink', '');
  const dripIsOn = get(fields, 'drip', false);
  const upperDripIsOn = get(fields, 'upperDrip', false);

  return (
    <div
      ref={refBlock => setRef(refBlock)}
      style={{ zIndex: z }}
      className={cx(
        styles['ImageDoubleText'],
        styles[colorClass],
        'drip-padding-on-children',
        { drip: dripIsOn, 'upper-drip': upperDripIsOn }
      )}
    >
      <div className="wh100 flex justify-center items-center overflow-hidden">
        <div
          className={cx(
            styles['ImageDoubleText--container'],
            {
              [styles[
                'ImageDoubleText--container--reverse-x'
              ]]: reverseArrangementX,
              [styles[
                'ImageDoubleText--container--reverse-y'
              ]]: reverseArrangementY,
              [styles['ImageDoubleText--container--reverse-xy']]:
                reverseArrangementY && reverseArrangementX
            },
            'w100 container-width flex items-center justify-between'
          )}
        >
          <div
            className={cx(
              styles['ImageDoubleText--image-container'],
              'circle square'
            )}
            style={{
              background: `url(${contentfulImgUtil(
                get(fields, 'image.fields.file.url', ''),
                '1400'
              )}) no-repeat center`,
              backgroundSize: 'cover'
            }}
          />
          <div
            className={cx(
              styles['ImageDoubleText--text-container'],
              'col-12 md-col-7 flex flex-column'
            )}
          >
            {smallTitle ? (
              <p className={`text-${smallTitleColor} small-title mb3`}>
                {smallTitle}
              </p>
            ) : null}
            {title && !title1 && !text1 && !title2 && !text2 ? (
              <h2
                className={cx(styles['ImageDoubleText__title'], 'carter mb2')}
              >
                {title}
              </h2>
            ) : null}
            {title1 || text1 ? (
              <div className="mb3">
                <h2 className="block-headline mb2">
                  {get(fields, 'title1', '')}
                </h2>
                <div
                  dangerouslySetInnerHTML={{
                    __html: marked(get(fields, 'text1', ''))
                  }}
                  className="markdown-block"
                />
              </div>
            ) : null}
            {title2 || text2 ? (
              <div>
                <h2 className="block-headline mb2">
                  {get(fields, 'title2', '')}
                </h2>
                <div
                  dangerouslySetInnerHTML={{
                    __html: marked(get(fields, 'text2', ''))
                  }}
                  className="markdown-block"
                />
              </div>
            ) : null}
            {buttonLabel ? (
              <Button
                variant="primary-responsive"
                className="inline-flex mt2"
                color="peach"
                label={buttonLabel}
                to={buttonLink}
              />
            ) : null}
          </div>
        </div>
      </div>
    </div>
  );
};

ImageDoubleText.propTypes = {
  z: PropTypes.number,
  block: PropTypes.shape({
    fields: PropTypes.shape({
      image: imageModel.propTypes,
      backgroundColor: PropTypes.string,
      reverseArrangementX: PropTypes.bool,
      reverseArrangementY: PropTypes.bool,
      smallTitle: PropTypes.string,
      smallTitleColor: PropTypes.string,
      title: PropTypes.string,
      title1: PropTypes.string,
      text1: PropTypes.string,
      title2: PropTypes.string,
      text2: PropTypes.string,
      buttonLabel: PropTypes.string,
      buttonLink: PropTypes.string,
      drip: PropTypes.bool
    })
  }),
  setRef: PropTypes.func
};

ImageDoubleText.defaultProps = {
  z: 1,
  block: {
    fields: {
      image: imageModel.default,
      backgroundColor: 'white',
      reverseArrangementX: false,
      reverseArrangementY: false,
      smallTitle: '',
      smallTitleColor: 'madison-blue',
      title: '',
      title1: '',
      text1: '',
      title2: '',
      text2: '',
      buttonLabel: '',
      buttonLink: '',
      drip: false
    }
  },
  setRef: () => {}
};

export default ImageDoubleText;
