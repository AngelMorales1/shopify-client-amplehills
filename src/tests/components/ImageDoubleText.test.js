import React from 'react';
import { shallow } from 'enzyme';

import ImageDoubleText from 'components/ImageDoubleText';

it('renders without data', () => {
  const component = shallow(<ImageDoubleText />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <ImageDoubleText
      z={0}
      block={{
        image: {
          sys: {
            space: {
              sys: {
                type: 'foo',
                linkType: 'foo',
                id: 'foo'
              }
            },
            id: 'foo',
            type: 'foo',
            createdAt: 'foo',
            updatedAt: 'foo',
            environment: { foo: {} },
            revision: 0,
            locale: 'foo'
          },
          fields: {
            title: 'foo',
            file: {
              url: 'foo',
              details: {
                size: 0,
                image: {
                  width: 0,
                  height: 0
                }
              },
              fileName: 'foo',
              contentType: 'foo'
            }
          }
        },
        backgroundColor: 'foo',
        title1: 'foo',
        title2: 'foo',
        text1: 'foo',
        text2: 'foo',
        title: 'foo'
      }}
    />
  );

  expect(component).toMatchSnapshot();
});
