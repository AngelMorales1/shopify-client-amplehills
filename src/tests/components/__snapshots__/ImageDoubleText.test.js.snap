// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ImageDoubleText
    block={
        Object {
            "backgroundColor": "foo",
            "image": Object {
              "fields": Object {
                "file": Object {
                  "contentType": "foo",
                  "details": Object {
                    "image": Object {
                      "height": 0,
                      "width": 0,
                    },
                    "size": 0,
                  },
                  "fileName": "foo",
                  "url": "foo",
                },
                "title": "foo",
              },
              "sys": Object {
                "createdAt": "foo",
                "environment": Object {
                  "foo": Object {},
                },
                "id": "foo",
                "locale": "foo",
                "revision": 0,
                "space": Object {
                  "sys": Object {
                    "id": "foo",
                    "linkType": "foo",
                    "type": "foo",
                  },
                },
                "type": "foo",
                "updatedAt": "foo",
              },
            },
            "text1": "foo",
            "text2": "foo",
            "title": "foo",
            "title1": "foo",
            "title2": "foo",
          }
    }
    z={0}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container-width flex items-center justify-between"
>
        <div
                className="circle square"
                style={
                        Object {
                                "background": "url() no-repeat center",
                                "backgroundSize": "cover",
                              }
                }
        />
        <div
                className="md-col-6"
        >
                <div
                        className="mb3"
                >
                        <h2
                                className="block-headline mb2"
                        />
                        <p
                                className="block-subheadline"
                        />
                </div>
                <div>
                        <h2
                                className="block-headline mb2"
                        />
                        <p
                                className="block-subheadline"
                        />
                </div>
        </div>
</div>,
      "className": "drip flex justify-center items-center",
      "style": Object {
        "zIndex": 0,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="circle square"
            style={
                        Object {
                                    "background": "url() no-repeat center",
                                    "backgroundSize": "cover",
                                  }
            }
/>,
          <div
            className="md-col-6"
>
            <div
                        className="mb3"
            >
                        <h2
                                    className="block-headline mb2"
                        />
                        <p
                                    className="block-subheadline"
                        />
            </div>
            <div>
                        <h2
                                    className="block-headline mb2"
                        />
                        <p
                                    className="block-subheadline"
                        />
            </div>
</div>,
        ],
        "className": "container-width flex items-center justify-between",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "circle square",
            "style": Object {
              "background": "url() no-repeat center",
              "backgroundSize": "cover",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="mb3"
>
                <h2
                                className="block-headline mb2"
                />
                <p
                                className="block-subheadline"
                />
</div>,
              <div>
                <h2
                                className="block-headline mb2"
                />
                <p
                                className="block-subheadline"
                />
</div>,
            ],
            "className": "md-col-6",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="block-headline mb2"
/>,
                  <p
                    className="block-subheadline"
/>,
                ],
                "className": "mb3",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-headline mb2",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-subheadline",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="block-headline mb2"
/>,
                  <p
                    className="block-subheadline"
/>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-headline mb2",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-subheadline",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container-width flex items-center justify-between"
>
          <div
                    className="circle square"
                    style={
                              Object {
                                        "background": "url() no-repeat center",
                                        "backgroundSize": "cover",
                                      }
                    }
          />
          <div
                    className="md-col-6"
          >
                    <div
                              className="mb3"
                    >
                              <h2
                                        className="block-headline mb2"
                              />
                              <p
                                        className="block-subheadline"
                              />
                    </div>
                    <div>
                              <h2
                                        className="block-headline mb2"
                              />
                              <p
                                        className="block-subheadline"
                              />
                    </div>
          </div>
</div>,
        "className": "drip flex justify-center items-center",
        "style": Object {
          "zIndex": 0,
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="circle square"
              style={
                            Object {
                                          "background": "url() no-repeat center",
                                          "backgroundSize": "cover",
                                        }
              }
/>,
            <div
              className="md-col-6"
>
              <div
                            className="mb3"
              >
                            <h2
                                          className="block-headline mb2"
                            />
                            <p
                                          className="block-subheadline"
                            />
              </div>
              <div>
                            <h2
                                          className="block-headline mb2"
                            />
                            <p
                                          className="block-subheadline"
                            />
              </div>
</div>,
          ],
          "className": "container-width flex items-center justify-between",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "circle square",
              "style": Object {
                "background": "url() no-repeat center",
                "backgroundSize": "cover",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="mb3"
>
                  <h2
                                    className="block-headline mb2"
                  />
                  <p
                                    className="block-subheadline"
                  />
</div>,
                <div>
                  <h2
                                    className="block-headline mb2"
                  />
                  <p
                                    className="block-subheadline"
                  />
</div>,
              ],
              "className": "md-col-6",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2
                      className="block-headline mb2"
/>,
                    <p
                      className="block-subheadline"
/>,
                  ],
                  "className": "mb3",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-headline mb2",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-subheadline",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2
                      className="block-headline mb2"
/>,
                    <p
                      className="block-subheadline"
/>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-headline mb2",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-subheadline",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ImageDoubleText
    block={
        Object {
            "fields": Object {
              "backgroundColor": "light-yellow",
              "image": Object {
                "fields": Object {
                  "file": Object {
                    "url": "",
                  },
                  "title": "",
                },
                "sys": Object {
                  "id": "",
                },
              },
              "text1": "",
              "text2": "",
              "title1": "",
              "title2": "",
            },
          }
    }
    z={1}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container-width flex items-center justify-between"
>
        <div
                className="circle square"
                style={
                        Object {
                                "background": "url() no-repeat center",
                                "backgroundSize": "cover",
                              }
                }
        />
        <div
                className="md-col-6"
        >
                <div
                        className="mb3"
                >
                        <h2
                                className="block-headline mb2"
                        />
                        <p
                                className="block-subheadline"
                        />
                </div>
                <div>
                        <h2
                                className="block-headline mb2"
                        />
                        <p
                                className="block-subheadline"
                        />
                </div>
        </div>
</div>,
      "className": "drip flex justify-center items-center",
      "style": Object {
        "zIndex": 1,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="circle square"
            style={
                        Object {
                                    "background": "url() no-repeat center",
                                    "backgroundSize": "cover",
                                  }
            }
/>,
          <div
            className="md-col-6"
>
            <div
                        className="mb3"
            >
                        <h2
                                    className="block-headline mb2"
                        />
                        <p
                                    className="block-subheadline"
                        />
            </div>
            <div>
                        <h2
                                    className="block-headline mb2"
                        />
                        <p
                                    className="block-subheadline"
                        />
            </div>
</div>,
        ],
        "className": "container-width flex items-center justify-between",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "circle square",
            "style": Object {
              "background": "url() no-repeat center",
              "backgroundSize": "cover",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="mb3"
>
                <h2
                                className="block-headline mb2"
                />
                <p
                                className="block-subheadline"
                />
</div>,
              <div>
                <h2
                                className="block-headline mb2"
                />
                <p
                                className="block-subheadline"
                />
</div>,
            ],
            "className": "md-col-6",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="block-headline mb2"
/>,
                  <p
                    className="block-subheadline"
/>,
                ],
                "className": "mb3",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-headline mb2",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-subheadline",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="block-headline mb2"
/>,
                  <p
                    className="block-subheadline"
/>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-headline mb2",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-subheadline",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container-width flex items-center justify-between"
>
          <div
                    className="circle square"
                    style={
                              Object {
                                        "background": "url() no-repeat center",
                                        "backgroundSize": "cover",
                                      }
                    }
          />
          <div
                    className="md-col-6"
          >
                    <div
                              className="mb3"
                    >
                              <h2
                                        className="block-headline mb2"
                              />
                              <p
                                        className="block-subheadline"
                              />
                    </div>
                    <div>
                              <h2
                                        className="block-headline mb2"
                              />
                              <p
                                        className="block-subheadline"
                              />
                    </div>
          </div>
</div>,
        "className": "drip flex justify-center items-center",
        "style": Object {
          "zIndex": 1,
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="circle square"
              style={
                            Object {
                                          "background": "url() no-repeat center",
                                          "backgroundSize": "cover",
                                        }
              }
/>,
            <div
              className="md-col-6"
>
              <div
                            className="mb3"
              >
                            <h2
                                          className="block-headline mb2"
                            />
                            <p
                                          className="block-subheadline"
                            />
              </div>
              <div>
                            <h2
                                          className="block-headline mb2"
                            />
                            <p
                                          className="block-subheadline"
                            />
              </div>
</div>,
          ],
          "className": "container-width flex items-center justify-between",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "circle square",
              "style": Object {
                "background": "url() no-repeat center",
                "backgroundSize": "cover",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="mb3"
>
                  <h2
                                    className="block-headline mb2"
                  />
                  <p
                                    className="block-subheadline"
                  />
</div>,
                <div>
                  <h2
                                    className="block-headline mb2"
                  />
                  <p
                                    className="block-subheadline"
                  />
</div>,
              ],
              "className": "md-col-6",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2
                      className="block-headline mb2"
/>,
                    <p
                      className="block-subheadline"
/>,
                  ],
                  "className": "mb3",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-headline mb2",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-subheadline",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2
                      className="block-headline mb2"
/>,
                    <p
                      className="block-subheadline"
/>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-headline mb2",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "",
                      "className": "block-subheadline",
                    },
                    "ref": null,
                    "rendered": "",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
